examples/client-logger/app-insights-logger/src/test/components/App.test.tsx

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { App } from "../../components";
+
+describe("App Insights Example App UI test", () => {
+	it("App renders", async (): Promise<void> => {
+		render(<App />);
+		await screen.findByText("Loading Shared container...");
+	});
+});
```

Comment:
It seems like `@testing-library/jest-dom` gives us DOM-related extensions for assertions, like `expect(<some-dom-node>).toBeDisabled()`... let's either use it here if relevant or remove the import (and the dependency altogether if we don't actually use it).